src/logger/types.ts

```diff
@@ -9,6 +9,12 @@ export enum LogContext {
   Notifications = 'notifications',
   ConversationAgent = 'conversation-agent',
   DMsAgent = 'dms-agent',
+
+  /**
+   * METRIC IS FOR INTERNAL USE ONLY, don't create any other loggers using this
+   * context
+   */
+  Metric = 'metric',
```

Comment:
This will be added as a Sentry tag and property `__context__` on all emitted events. In dev, it also shows in the console like `(metric) <name>`.